src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -250,6 +250,19 @@
                         [hideLabelMobile]="false"
                         [routerLink]="['/courses', courseId, 'exercises', exercise.type + '-exercises', exercise.id, 'participate', gradedParticipation!.id]"
                     ></button>
+                    @if (exercise.allowFeedbackRequests && athenaEnabled && exercise.type === ExerciseType.TEXT) {
+                        <button
+                            class="btn btn-primary"
+                            (click)="requestFeedback()"
+                            [class.btn-sm]="smallButtons"
+                            [disabled]="gradedParticipation?.submissions?.length == 1 && !gradedParticipation!.submissions![0].submitted"
```

Comment:
Why is it only disabled for exactly one submission? What happens if a student submits twice?

Reply:
I made it more general now for the last submission and also disabled if feedback is being generated